src/Command/AddMassMediaCommand.php

```diff
@@ -120,7 +122,10 @@ protected function insertMedia(array $data, OutputInterface $output)
     protected function optimize()
     {
         if (null !== $this->managerRegistry) {
-            $this->managerRegistry->getManager()->getUnitOfWork()->clear();
+            $manager = $this->managerRegistry->getManager();
+
+            \assert($manager instanceof DocumentManagerInterface || $manager instanceof EntityManagerInterface);
+            $manager->getUnitOfWork()->clear();
```

Comment:
I think we introduce a ClearableManagerInterface, couldn't we do an instanceof check instead ?

https://github.com/sonata-project/sonata-doctrine-extensions/pull/323/files